Remotion/Mixins/Samples/DynamicMixinBuilding/UnitTests/DynamicMixinBuilderTest.cs

```diff
@@ -109,14 +110,15 @@ private IPipelineRegistry CreatePipelineRegistry ()
 
     private string PrepareDirectory ()
     {
-      string directory = Path.Combine(Environment.CurrentDirectory, "DynamicMixinBuilder.Generated");
-      if (Directory.Exists(directory))
-        Directory.Delete(directory, true);
-      Directory.CreateDirectory(directory);
-
-      CopyFile(typeof(Mixin<,>).Assembly.ManifestModule.FullyQualifiedName, directory); // Core/Mixins assembly
-      CopyFile(typeof(MethodInvocationHandler).Assembly.ManifestModule.FullyQualifiedName, directory); // Samples assembly
-      return directory;
+      var tempDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
```

Comment:
I would suggest not changing the variable's name. it introduces noise to the changeset and doesn't help us better understand the test's semantics.